jobs/JOBSPROC_RAP_DUMP_POST

```diff
@@ -142,11 +144,11 @@ export USHobsproc=${USHobsproc:-$HOMEobsproc/ush}
 # --------------------------------------------
 # Paths for bufr_dumplist items
 # --------------------------------------------
-export HOMEbufr_util=${HOMEbufr_util:\
--${NWROOT:?}/bufr_util.${bufr_util_ver:?\
-"###FATAL ERROR \"bufr_util_ver\" is not set"}}
+export HOMEbufr_dump=${HOMEbufr_dump:\
+-${PACKAGEROOT:?}/bufr_dump.${bufr_dump_ver:?\
+"###FATAL ERROR \"bufr_dump_ver\" is not set"}}
```

Comment:
See same as JOBSPROC_RAP_DUMP. These three lines are redundant and not required. $HOMEbufr_dump is already predefined when bufr_dump/1.0.0  module is loaded. I suggest you remove these lines.